src/Protocol/ActivityPub/Processor.php

```diff
@@ -58,7 +58,7 @@ class Processor
 	 *
 	 * @return string converted body
 	 */
-	private static function convertMentions($body)
+	private static function convertPleromaMentions($body)
```

Comment:
Why "Pleroma"? Are you sure that it only is used for Pleroma?

Reply:
It will be applied to all incoming messages, but the comment of the method itself hints at Pleroma.

For mentions where the link doesn't include the mention character, this method doesn't do anything.

Reply:
The function is for mentions that are in a structure that Pleroma uses - but the function will work for other systems as well. So I think it shouldn't have the name "Pleroma" in it.